test/ansible/cleanup.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
The idiomatic name for the folder would be  "playbooks", or put the playbooks yaml along with the "roles" folder.
https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#directory-layout